Makefile

```diff
@@ -26,9 +26,26 @@ fmt:
 .PHONY: lt
 lt: lint test
 
+.PHONY: devnet-up
+devnet-up:
+	@kurtosis run \
+			--enclave eth-devnet \
+		github.com/kurtosis-tech/ethereum-package \
```

Comment:
GitHub.com/flashbots/ethereum-package

Reply:
Sorry, forgot to add more context. That package did not work for me initially, I ended using `gitHub.com/flashbots/flashbots/kurtosis-ethereum-package` and it worked.

Reply:
that's weird.

in my tests it I would use `github.com/flashbots/flashbots/kurtosis-ethereum-package` here, then I get the error:

```
There was an error interpreting Starlark code 
Error in cloning git repository 'https://github.com/flashbots/flashbots.git' to '/kurtosis-data/tmp-repositories/tmp-repo-dir-1070831636/flashbots/flashbots'. Make sure that 'https://github.com/flashbots/flashbots.git' exists or if it's a private repository, that you are logged into GitHub via `kurtosis github login`.
```

.. and if I use `github.com/flashbots/kurtosis-ethereum-package` then the error is:

```
There was an error interpreting Starlark code 
Error occurred while validating /kurtosis-data/repositories/flashbots/kurtosis-ethereum-package/kurtosis.yml
        Caused by: The package name in kurtosis.yml must match the location it is in. Package name is 'github.com/kurtosis-tech/ethereum-package' and kurtosis.yml is found here: 'github.com/flashbots/kurtosis-ethereum-package'
```

previously this line was referring to the relative path (with the assumption that `https://github.com/flashbots/kurtosis-ethereum-package` repo would be cloned _next_ to the `suapp-examples` repo), in which case kurtosis would not do the package name check.  but after the PR I made (https://github.com/kurtosis-tech/ethereum-package/pull/489) got merged using locally cloned repo was no longer necessary.

a.t.m. it all does work Ok on my machine.  does it work on yours now?